src/flags.ads

```diff
@@ -158,6 +158,10 @@ package Flags is
    --  If True, disp original source line and a caret indicating the column.
    Flag_Caret_Diagnostics : Boolean := False;
 
+   -- If True, warning is printed for each uncovered PSL cover points at the
+   -- end of simulation
+   Flag_Psl_Warn_Uncovered : Boolean := False;
```

Comment:
It seems that this flag decides whether to report or not, instead of selecting the severity level (warning or other). Therefore, I would suggest `Flag_Psl_Report_Uncovered`.